### ISSUE TITLE
Tags don't accept commas in the value.

### ISSUE THREAD

**Issue by griznog** [hpcng/warewulf#413]
Not sure if this is by design, but I'm unable to set a tag to a value that contains a comma, for example:

```
[I am root!@frankie:nhc]# wwctl node set --yes --tag "mytag=myvalue" cpu-a-001
[I am root!@frankie:nhc]# wwctl node set --yes --tag "mytag=myvalue,commabreaksit" cpu-a-001
panic: runtime error: index out of range [1] with length 1

goroutine 1 [running]:
github.com/hpcng/warewulf/internal/app/wwctl/node/set.CobraRunE(0x16b9f00, 0xc00047ca00, 0x1, 0x4, 0x0, 0x0)
	/hpc/admin/Warewulf/warewulf/internal/app/wwctl/node/set/main.go:319 +0x753f
github.com/spf13/cobra.(*Command).execute(0x16b9f00, 0xc00047c9c0, 0x4, 0x4, 0x16b9f00, 0xc00047c9c0)
	/hpc/admin/Warewulf/warewulf/vendor/github.com/spf13/cobra/command.go:850 +0x472
github.com/spf13/cobra.(*Command).ExecuteC(0x16b5ac0, 0x300000002, 0xc000000180, 0xc0000ac740)
	/hpc/admin/Warewulf/warewulf/vendor/github.com/spf13/cobra/command.go:958 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
	/hpc/admin/Warewulf/warewulf/vendor/github.com/spf13/cobra/command.go:895
main.main()
	/hpc/admin/Warewulf/warewulf/cmd/wwctl/main.go:13 +0x45
```

**Comment by mslacken** [comment]
Yes, I think this is due to cobra. A commas are recognized as string separators, but a tag is of type `string` and node `[]string`

**Comment by griznog** [comment]
I was hoping to set a tag for replicated NFS servers in automount maps, like `nfsserverlist=172.16.0.1(1),172.16.0.2(2)` and then pull that into an autofs map template. Can be worked around in the template, but it's a lot uglier than just having the value have a comma in it.

**Comment by mslacken** [comment]
This is resolved in 4.3.0 as there is no `tag` command any more, but `taggadd ` behaves the same way.
Also `wwct [node|profile| edit` 
does not show this problem.
